src/components-examples/material/tooltip/index.ts

```diff
@@ -1,5 +1,6 @@
 export {TooltipAutoHideExample} from './tooltip-auto-hide/tooltip-auto-hide-example';
 export {TooltipCustomClassExample} from './tooltip-custom-class/tooltip-custom-class-example';
+export {TooltipDefaultCustomClassExample} from './tooltip-default-custom-class/tooltip-default-custom-class-example';
```

Comment:
Docs are good, but this option is pretty basic so I'm not sure that we need a live example for it.

Reply:
Got it. I removed the example.